docs/source/datasets/index.rst

```diff
@@ -172,6 +172,63 @@ about CPS, please visit the `CPS homepage <https://www.census.gov/programs-surve
 
 WIC
 ---
+The Special Supplemental Nutrition Program for Women, Infants, and Children (WIC) is a government benefits program designed to support mothers and young
+children. The main qualifications are income and the presence of young children in the home. To find out more about this service, please visit the `WIC 
+homepage <https://www.fns.usda.gov/wic>`_.
```

Comment:
We need to explain how our simulated data relates to this program. For example:

Pseudopeople can generate a simulated version of the administrative data that would be recorded by WIC.
This is a yearly file of information about all simulants enrolled in the program as of the end of that year.